<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>votingMethod</class>
 <widget class="QWidget" name="votingMethod">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>1300</x>
     <y>630</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>вернуться</string>
   </property>
  </widget>
  <widget class="QLabel" name="test_image">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>70</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Тестовая фотография</string>
   </property>
  </widget>
  <widget class="QLabel" name="histogram_answer">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ответ гистограммы</string>
   </property>
  </widget>
  <widget class="QLabel" name="scaling_answer">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>420</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ответ скейлинга</string>
   </property>
  </widget>
  <widget class="QLabel" name="random_answer">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>420</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ответ рандома</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbph_answer">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>420</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ответ LBPH</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextImageButton">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>500</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Далее</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prevImageButton">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>560</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ответ гистограмм</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>370</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ответ скейлинга</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>370</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ответ рандомных точек</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>370</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ответ LBPH</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>30</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Тестовое изображение</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
